streaming/redis.js

```diff
@@ -4,44 +4,110 @@ import { parseIntFromEnvValue } from './utils.js';
 
 /**
  * @typedef RedisConfiguration
- * @property {import('ioredis').RedisOptions} redisParams
- * @property {string} redisPrefix
  * @property {string|undefined} redisUrl
+ * @property {import('ioredis').RedisOptions} redisOptions
  */
 
+/**
+ *
+ * @param {NodeJS.ProcessEnv} env
+ * @returns {boolean}
+ */
+function hasSentinelConfiguration(env) {
+  return (
+    typeof env.REDIS_SENTINELS === 'string' &&
+    env.REDIS_SENTINELS.length > 0 &&
+    typeof env.REDIS_SENTINEL_MASTER === 'string' &&
+    env.REDIS_SENTINEL_MASTER.length > 0
```

Comment:
I think we want to require both `REDIS_SENTINELS` and `REDIS_SENTINEL_MASTER` to enable sentinel mode, rather than setting a default master name like `mymaster` which was originally in #26571.

Reply:
I think the idea came from the sentinel docs always using "mymaster" as an example. And since this not a hostname but just an identifier, it could be assumed that many will just copy those examples.

But I agree, blindly copying examples and not using sensible identifiers is not something we need to support :slightly_smiling_face: